sys/net/application_layer/dhcpv6/client.c

```diff
@@ -208,7 +206,20 @@ static inline bool _is_tid(dhcpv6_msg_t *msg)
 
 static inline uint32_t _now_cs(void)
 {
+#if IS_USED(MODULE_ZTIMER)
+    return (uint32_t)(ztimer_now(ZTIMER_MSEC) / US_PER_CS);
+#else
     return (uint32_t)(xtimer_now_usec64() / US_PER_CS);
+#endif
+}
+
+static inline uint32_t _now_sec(void)
+{
+#if IS_USED(MODULE_ZTIMER)
+    return (uint32_t)ztimer_now(ZTIMER_MSEC);
```

Comment:
needs a factor of 1/1000?
```suggestion
    return (uint32_t)(ztimer_now(ZTIMER_MSEC) / MS_PER_SEC));
```

Reply:
I think that was supposed to be
```suggestion
    return (uint32_t)ztimer_now(ZTIMER_SEC);
```